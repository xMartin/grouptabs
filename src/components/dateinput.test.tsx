import renderer from "react-test-renderer";
import DateInput from "./dateinput";

it("renders prefilled input", () => {
  const tree = renderer
    .create(<DateInput date="2020-03-22" onChange={vi.fn()} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
