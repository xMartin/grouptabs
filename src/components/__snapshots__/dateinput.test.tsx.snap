// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders prefilled input 1`] = `
<div
  className="form-row-input date-input"
>
  <button
    className=""
    onClick={[Function]}
    type="button"
  >
    today
  </button>
  <button
    className=""
    onClick={[Function]}
    type="button"
  >
    yesterday
  </button>
  <input
    onChange={[Function]}
    type="date"
    value="2020-03-22"
  />
</div>
`;
